### DISCUSSION TITLE
articles/programmer-reading-list-1/

### DISCUSSION THREAD

**Discussion by giscus** [piglei/piglei-giscus-comments#12] [General]
# articles/programmer-reading-list-1/

piglei 的 blog

https://www.piglei.com/articles/programmer-reading-list-1/

**Comment by east4ming** [comment]
高质量软件: 电报; 低质量软件: 小而美.
确实证明了: "但实际上，在内在质量上投入并不增加成本，反而能降低整体花费。"

**Comment by gongchangkai** [comment]
啥时候推更完😄

**Comment by AlphaChen** [comment]
都是好文
